.github/workflows/managed.yml

```
@@ -120,3 +120,11 @@ jobs:
           env | sort
           go env | sort
           git status
+          docker exec pmm-server env
```

```suggestion
          docker exec pmm-server env | sort
```
nit: it would make it easier to find a variable of interest

Same for `go env`